setup.cfg

```diff
@@ -0,0 +1,7 @@
+[isort]
+known_first_party = cabot
+known_third_party = celery,jenkins,freezegun,twilio
+multi_line_output = 0
```

Comment:
@frankh You are okay with that format ? ;-D

Reply:
yep 👍